oar-lps/libs/oarlps/src/lib/shared/searchfields-list/searchfields-list.service.ts

```diff
@@ -33,7 +33,7 @@ export class SearchfieldsListService {
      */
     get(): Observable<any> {
         //   console.log("Getting fields from:", this.RMMAPIURL + 'fields');
-    return this.http.get(this.RMMAPIURL + 'records/fields');
+    return this.http.get(this.RMMAPIURL + 'fields');
```

Comment:
This should actually be `'records/fields'`.

Reply:
mdSearch already has "records" at the end:
"mdSearch": "https://data.nist.gov/rmm/records/",
If I append "records/fields", the URL will be "https://data.nist.gov/rmm/records/records/fields" which will gave me 404 error.
Should I remove "records" from mdSearch?

Reply:
Hmm, let me check...